<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDTSettingDlg</class>
 <widget class="QDialog" name="CDTSettingDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>728</width>
    <height>693</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Setting Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,1,1,1,0">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="txtProtocolType">
       <property name="text">
        <string>Protocol Type:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbbProtocol">
       <item>
        <property name="text">
         <string>Standard CDT</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>GC Interace CDT</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>UT CDT</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>NR CDT</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>NR UDP</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Control Byte:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edControlType"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnReset">
       <property name="text">
        <string>Restore Default Settings</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxYx">
     <property name="title">
      <string>YX</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="3">
       <widget class="QSpinBox" name="edYxTime">
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="suffix">
         <string>ms</string>
        </property>
        <property name="maximum">
         <number>999999999</number>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>YX Number:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_19">
        <property name="text">
         <string>Cycle Send Time:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Start IO:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Func Code:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="edYxType"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="edYxStartIo"/>
      </item>
      <item row="0" column="5">
       <widget class="QLineEdit" name="edYxNum">
        <property name="text">
         <string notr="true">10</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Frame Type:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="edYxFuncode"/>
      </item>
      <item row="2" column="1" colspan="5">
       <widget class="QLabel" name="txtYxSheetInfo">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="btnYxImportSheet">
        <property name="text">
         <string>Import YX Point File</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxYc">
     <property name="title">
      <string>YC</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="4">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>YC Number:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Start IO:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_20">
        <property name="text">
         <string>Cycle Send Time:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Func Code:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QSpinBox" name="edYcTime">
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="suffix">
         <string>ms</string>
        </property>
        <property name="maximum">
         <number>999999999</number>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="edYcFuncode"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="edYcStartIo"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Frame Type:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="edYcType"/>
      </item>
      <item row="0" column="5">
       <widget class="QLineEdit" name="edYcNum">
        <property name="text">
         <string notr="true">10</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="5">
       <widget class="QLabel" name="txtYcSheetInfo">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="btnYcImportSheet">
        <property name="text">
         <string>Import YC Point File</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxVYx">
     <property name="title">
      <string>Virtual YX</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="1" column="0">
       <widget class="QLabel" name="label_18">
        <property name="text">
         <string>Start IO:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>Virtual YX Number:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>Frame Type:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_21">
        <property name="text">
         <string>Cycle Send Time:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>Func Code:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QSpinBox" name="edVYxTime">
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="suffix">
         <string>ms</string>
        </property>
        <property name="maximum">
         <number>999999999</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="edVYxType"/>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="edVYxFuncode"/>
      </item>
      <item row="0" column="5">
       <widget class="QLineEdit" name="edVYxNum">
        <property name="text">
         <string notr="true">10</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="edVYxStartIo"/>
      </item>
      <item row="2" column="1" colspan="5">
       <widget class="QLabel" name="txtVYxSheetInfo">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="btnVYxImportSheet">
        <property name="text">
         <string>Import Virtual YX Point File</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>YK</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="2">
       <widget class="QLineEdit" name="edYkReqType"/>
      </item>
      <item row="2" column="3">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Control Close:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="edYkReqCode"/>
      </item>
      <item row="3" column="3">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>Locked:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Control Open:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QLineEdit" name="edYkOpen"/>
      </item>
      <item row="3" column="2">
       <widget class="QLineEdit" name="edYkUnlock"/>
      </item>
      <item row="0" column="4">
       <widget class="QLineEdit" name="edYkAckType"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>UnLocked:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_23">
        <property name="text">
         <string>Request Func Code:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Request Frame Type:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_22">
        <property name="text">
         <string>Response Frame Type:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="edYkClose"/>
      </item>
      <item row="3" column="4">
       <widget class="QLineEdit" name="edYkLock"/>
      </item>
      <item row="1" column="4">
       <widget class="QLineEdit" name="edYkAckCode"/>
      </item>
      <item row="1" column="5">
       <widget class="QLabel" name="label_25">
        <property name="text">
         <string>Exe Func Code:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="label_24">
        <property name="text">
         <string>Response Func Code:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="QLineEdit" name="edYkExeCode"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnOk">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cbbProtocol</tabstop>
  <tabstop>edControlType</tabstop>
  <tabstop>btnReset</tabstop>
  <tabstop>edYxType</tabstop>
  <tabstop>edYxFuncode</tabstop>
  <tabstop>edYxNum</tabstop>
  <tabstop>edYxStartIo</tabstop>
  <tabstop>edYxTime</tabstop>
  <tabstop>edYcType</tabstop>
  <tabstop>edYcFuncode</tabstop>
  <tabstop>edYcNum</tabstop>
  <tabstop>edYcStartIo</tabstop>
  <tabstop>edYcTime</tabstop>
  <tabstop>edVYxType</tabstop>
  <tabstop>edVYxFuncode</tabstop>
  <tabstop>edVYxNum</tabstop>
  <tabstop>edVYxStartIo</tabstop>
  <tabstop>edVYxTime</tabstop>
  <tabstop>edYkReqType</tabstop>
  <tabstop>edYkAckType</tabstop>
  <tabstop>edYkReqCode</tabstop>
  <tabstop>edYkAckCode</tabstop>
  <tabstop>edYkExeCode</tabstop>
  <tabstop>edYkClose</tabstop>
  <tabstop>edYkOpen</tabstop>
  <tabstop>edYkUnlock</tabstop>
  <tabstop>edYkLock</tabstop>
  <tabstop>btnOk</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
